development/build/static.js

```diff
@@ -15,11 +15,21 @@ module.exports = function createStaticAssetTasks({
   livereload,
   browserPlatforms,
   shouldIncludeLockdown = true,
+  shouldIncludeSnow = true,
   buildType,
 }) {
-  const [copyTargetsProd, copyTargetsDev] = getCopyTargets(
-    shouldIncludeLockdown,
-  );
+  const copyTargetsProds = {};
+  const copyTargetsDevs = {};
+
+  browserPlatforms.forEach((browser) => {
+    const [copyTargetsProd, copyTargetsDev] = getCopyTargets(
+      shouldIncludeLockdown,
+      // Snow currently only works on Chromium based browsers
+      shouldIncludeSnow && browser === 'chrome',
```

Comment:
make this field an obj so we can reuse it later `{ snow: shouldIncludeSnow && browser === 'chrome', }`

Reply:
I was following the exact same pattern as the other `shouldIncludeLockdown` boolean, isn't that the same?

Reply:
Do you mean combine both `shouldIncludeLockdown` and `shouldIncludeSnow` into an object?

Reply:
ah, i missed that there was already a condition here - fine to keep as is